app/employee/paystubs/page.tsx

```
@@ -1,185 +1,376 @@
+}: {
+  stub: ProcessedPaystub;
+  isExpanded: boolean;
+  onToggle: () => void;
+}) {
+  return (
+    <Card className="shadow-md overflow-hidden pt-0">
+      <div
+        role="button"
+        onClick={onToggle}
+        className="w-full bg-green-900 text-white px-6 py-4 hover:bg-green-800 transition-colors flex justify-between items-center cursor-pointer"
+      >
```

Biome (2.4.10)

[error] The HTML element with the interactive role "button" is not focusable.

(lint/a11y/useFocusableInteractive)